### ISSUE TITLE
training data

### ISSUE THREAD

**Issue by Holmes-GU** [polimi-ispl/icpr2020dfdc#29]
Thanks for sharing codes and sorry for disturbing you.

When I downloading DFDC-preview dataset (5k videos), I fail to find it and its json file, dataset.json. The name of each video is like pyfnfvsxez.mp4 and is different from the format 1255229_1003254_A_001.mp4. How can I download the same datasets of yours?

**Comment by CrohnEngineer** [comment]
Hey @Holmes-GU ,

You need to download the full DFDC dataset, not the DFDC-preview. You can find it here https://www.kaggle.com/c/deepfake-detection-challenge/data .
Please read all throughout the README.md to have a general idea of what you need and how to run our code.
Bests,

Edoardo

**Comment by Holmes-GU** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        Actually， I saw your comments at 'https://github.com/NVIDIA/DALI/issues/1599'. The comments on 3 Jan 2020 shows the direction: /nas/public/dataset/fb_dfd_release_0.1_final/method_A/1795659/1795659_B/2059066_1795659_B_001.mp4 1. This direction is format of DFDC-preview, where can I download it? It's very important to. Thank you.
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                ellery-holmes
                            
                            
                                
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 4/2/2021 14:04，Edoardo Daniele ***@***.***> wrote： 



Hey @Holmes-GU ,
You need to download the full DFDC dataset, not the DFDC-preview. You can find it here https://www.kaggle.com/c/deepfake-detection-challenge/data .
Please read all throughout the README.md to have a general idea of what you need and how to run our code.
Bests,
Edoardo

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by CrohnEngineer** [comment]
Hey @Holmes-GU ,

At the time of writing that comment we didn't developed the pipeline used at last for partecipating in the challenge and obtaining the results presented in the paper.
You can download the preview dataset here https://ai.facebook.com/datasets/dfdc/ , but keep in mind that the code as it is won't work for this dataset and we are not planning to add support for the DFDC-preview.
Bests,

Edoardo

**Comment by Holmes-GU** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    Hi, thanks for the response! Actually, the video files downloaded from https://ai.facebook.com/datasets/dfdc/ are no longer the same as the description in the dfdc preview paper. The downloaded format is a series of alphabetas and mp4 suffix, like "zzrglepohd.mp4". However, the described format should be "ID1_ID2_method_index.mp4". We guess the dataset has been updated and maybe only the filenames are changed. Do you have the dataset of which the format is consistent with the paper? If so, could you please give us the video file names with their md5sum of your downloaded DFDC Preview dataset? Appreciate in advance.
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                ellery-holmes
                            
                            
                                
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 4/2/2021 14:32，Edoardo Daniele ***@***.***> wrote： 



Hey @Holmes-GU ,
At the time of writing that comment we didn't developed the pipeline used at last for partecipating in the challenge and obtaining the results presented in the paper.
You can download the preview dataset here https://ai.facebook.com/datasets/dfdc/ , but keep in mind that the code as it is won't work for this dataset and we are not planning to add support for the DFDC-preview.
Bests,
Edoardo

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by nicobonne** [comment]
Here's the md5sum of each video in the preview.

[dfdc_preview_md5sum.txt](https://github.com/polimi-ispl/icpr2020dfdc/files/6248336/dfdc_preview_md5sum.txt)

**Comment by Holmes-GU** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        Thanks for the response again. We checked the md5sum from your "dfdc_preview_md5sum.txt" with our local videos files, but only to find that the md5sum values shared no common. We tried to contact the dataset authors days ago but still got no respones. I know the whole dataset is quite big in filesize, but would you kindly considering sharing it with us? Like sending through web services such as Dropbox, GoogleDrive? If so, we may discuss the delivery form and details by email and my private email is ***@***.*** Looking forward to your response and appreciate in advance.
            
        
    
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                ellery-holmes
                            
                            
                                
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 4/2/2021 15:47，Nicolò ***@***.***> wrote： 



Here's the md5sum of each video in the preview.
dfdc_preview_md5sum.txt

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by nicobonne** [comment]
I'm in the process of uploading the dataset on Dropbox, which is the only cloud provider we can use. I'll let you know when the uploading is done, but we can't keep 50 GB in the cloud forever, so I'll remove it as soon as you download it. In your previous message your email got obscurate, could you please contact me at nicolo.bonettini@polimi.it?

**Comment by nicobonne** [comment]
@Holmes-GU

**Comment by Holmes-GU** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        Thank you. I have sent a message ***@***.***
    
        
            
        
    
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                Zhihao Gu
                            
                            
                                
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 
    

    On 04/14/2021 16:18，Nicolò ***@***.***> wrote： 



@Holmes-GU

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by bysen32** [comment]
@nicobonne Hello, I really need DFDC and DFDC Preview test datasets.  I have tried for a long time but I have been unable to register an account on dfdc.ai. If it is convenient, could you provide a cloud storage download address?  Thank you very much.
